src/app/LoginController.java

```diff
@@ -0,0 +1,64 @@
+package app;
+
+import java.io.IOException;
+import java.net.URL;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.ResourceBundle;
+import java.util.Timer;
+import java.util.TimerTask;
+import javafx.application.Platform;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Label;
+import javafx.scene.control.PasswordField;
+import javafx.scene.control.TextField;
+import javafx.scene.paint.Color;
+
+/**
+ * FXML Controller class
+ *
+ * @author maximdumont
+ */
+public class LoginController implements Initializable {
+    @FXML private TextField username;
+    @FXML private PasswordField password;
+    @FXML private Label errorLabel;
+    @FXML private Label clockLabel;
+    /**
+     * Initializes the controller class.
+     */
+    @Override
+    public void initialize(URL url, ResourceBundle rb) {
+        startClock();
+    }    
+    
+        private void startClock(){
+        String timeStamp = new SimpleDateFormat("HH:mm").format(Calendar.getInstance().getTime());
```

Comment:
Please indent. You can Ctrl-shift-F or Alt-shift-F (depending on platform) to make that happen. This isn't a blocker; please just fix up the formatting in your files, commit, and push before merging to master.